packages/server/src/payloads.ts

```diff
@@ -180,13 +180,15 @@ export const buildEpicData = async (
 
     const { test, variant } = result.result;
 
+    const choiceCardAmounts = await cachedChoiceCardAmounts();
     const tickerSettings = await getTickerSettings(variant);
     const showReminderFields = getReminderFields(variant);
 
-    const variantWithTickerAndReminder = {
+    const variantWithTickerAndReminderAndChoiceCardAmounts = {
```

Comment:
I wonder if `propsVariant` is a nicer name now?
We're essentially augmenting the `variant` from the test with extra data (unrelated to the test) needed by the client

Reply:
✅